datafusion/src/optimizer/filter_push_down.rs

```diff
@@ -368,6 +381,7 @@ fn optimize(plan: &LogicalPlan, mut state: State) -> Result<LogicalPlan> {
                     new_filters.push(filter_expr.clone());
                 }
             }
+            println!("{:?}", &state);
```

Comment:
forgot to delete this? :P

Reply:
Thanks, yes, old fashioned debugging 😆

Reply:
lol, we are in the same camp, I basically only use print for debugging except for rare kernel bugs :D